samples/client/petstore/python-experimental/test/__init__.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+
+# flake8: noqa
+"""
+    OpenAPI Petstore
+
+    This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from __future__ import absolute_import
+
+__version__ = "1.0.0"
+
+# import models into model package
+#from test.test_additional_properties_any_type import Testadditional_properties_any_type.AdditionalPropertiesAnyType
```

Comment:
Please remove this file from python-experimental because we are not yet using it